Makefile

```diff
@@ -140,29 +140,37 @@ REG?=$(ORG)/
 TAG?=devel
 export TAG
 
-e2e-fpga:
-	@$(GO) test -v ./test/e2e/... -ginkgo.v -ginkgo.show-node-events -ginkgo.focus "FPGA" -delete-namespace-on-failure=false
+generate-ginkgo-opt:
```

Comment:
generate-ginkgo-opt: this one?
I thought it would be useful to combine together with E2E_LEVEL (that has only `full` and `basic`)..
But, if you think it is not really useful, then let me drop it.

Reply:
Umm, let me tell you why I thought it is useful.

For example, many times I wanted to test of only one app of one resource.

In case of running tests for multiple devices, we need to just use `go test`, but in case of testing only one device for specific app, i was thinking it would be easy if we can use `make e2e-something`

Reply:
If you still don't buy it,, let me drop it..! (^^);

Reply:
I don't understand why a separate target is needed. You can set env variables without it too, can't you?

Reply:
Ah, sorry, I was really stupid. Yeah, true. Idk either why I made the target.. I dropped.!